### PR TITLE
allow redo search after article searches

### PR DESCRIPTION
**Pull request by joewalsh**

### CODE REVIEW

File: Wikipedia/Code/PlacesViewController.swift [hunk 1]
```diff
@@ -167,7 +167,7 @@ class PlacesViewController: UIViewController, MKMapViewDelegate, UISearchBarDele
     func mapView(_ mapView: MKMapView, regionDidChangeAnimated animated: Bool) {
         _mapRegion = mapView.region
         guard performDefaultSearchOnNextMapRegionUpdate == false else {
-            performDefaultSearch(withRegion: mapView.region)
+            performDefaultSearchIfNessary(withRegion: mapView.region)
```
Comment (amrox) [inline Wikipedia/Code/PlacesViewController.swift]: Nessary -> Necessary

Comment (montehurd) [inline Wikipedia/Code/PlacesViewController.swift]: typo

**Comment by montehurd** [comment]
Small typo.

**Comment by joewalsh** [comment]
I should add a spell check to my pre-commit hook

**Pull request merged**